src/engine/Operation.h

```diff
@@ -228,21 +243,14 @@ class Operation {
   //! Compute the result of the query-subtree rooted at this element..
   virtual ResultTable computeResult() = 0;
 
-  // Create and store the complete runtime information for this operation after
-  // it has either been succesfully computed or read from the cache.
-  virtual void updateRuntimeInformationOnSuccess(
-      const ConcurrentLruCache::ResultAndCacheStatus& resultAndCacheStatus,
-      size_t timeInMilliseconds) final;
-
   // Similar to the function above, but the components are specified manually.
   // If nullopt is specified for the last argument, then the `_runtimeInfo` is
   // expected to already have the correct children information. This is only
   // allowed when `cacheStatus` is `cachedPinned` or `cachedNotPinned`,
   // otherwise a runtime check will fail.
   virtual void updateRuntimeInformationOnSuccess(
       const ResultTable& resultTable, ad_utility::CacheStatus cacheStatus,
```

Comment:
Le't see if this works as you intended, I think for cached results we actually still need to overwrite the children.